src/sentry/plugins/sentry_mail/models.py

```diff
@@ -162,7 +162,12 @@ def get_send_to(self, project, event=None):
                         is_active=True,
                         sentry_orgmember_set__organizationmemberteam__team__id__in=teams_to_resolve,
                     ).values_list('id', flat=True)
-                return send_to_list
+
+                alert_settings = project.get_member_alert_settings(self.alert_option_key)
+                disabled_users = set(
+                    user for user, setting in alert_settings.items() if setting == 0
+                )
+                return set([user for user in send_to_list if user not in disabled_users])
```

Comment:
Hmm.. this is a bit unrelated but, should `send_to_list` and `teams_to_resolve` have been sets instead of lists? I was wondering why not just take the difference between `disabled_users` and `send_to_list`, but I see that's a list as the name suggests. Just curious.

Reply:
Yeah I think it's slightly nicer to use sets, so made the change.